src/menus/EdgingSettingsMenu.cpp

```diff
@@ -175,6 +227,10 @@ static void buildMenu(UIMenu *menu) {
   menu->addItem(&MinimumOnTimeInput);
   menu->addItem(&ArousalLimitInput);
   menu->addItem(&SensorSensitivityInput);
+  menu->addItem(&ClenchPressureSensitivity); //  Clench Menu
+  menu->addItem(&ClenchTimeThreshold);       //  Clench Menu
+  menu->addItem(&EdgingDuration);            //   Auto Edging
+  menu->addItem(&PostOrgasmDuration);        //   Post Orgasm
```

Comment:
There's so many features being added here, we might need to make a submenu. But not right now.